docs/conf/au/2017/news/event-recap-feedback

```diff
@@ -0,0 +1,143 @@
+:template: 2017/au-content.html
+
+.. post:: Dec 1, 2017
+   :tags: 2017, australia
+
+Write the Docs Day Australia 2017 - Recap
+=================================================
+
+A big thank you to all attending the 2017 and our first Australian Write the
+Docs Day  in Melbourne. We are extremely grateful to our sponsors, volunteers,
+speakers and attendees. We loved the time, effort and passion put in by
+everyone for this event.
+
+Hopefully, this event met people’s expectations and provided an opportunity to
+learn, meet and  interact with other documentarians from a growing community.
```

Comment:
I generally go for a more "everyone we talked to said this event met and exceeded their expectations!" vibe on these, but I'll leave it up to you. If you're going for humble, I can dig it :)